3-arrays/properties_and_methods_of_arrays.js

```diff
@@ -0,0 +1,74 @@
+// Some properties and methods of array are :
+
+
+// 1. Returns the number of elements :
+
+		//length
+
+// example
+
+var fruits = ["Orange", "Apple", "Banana"];
+
+fruits.length=3;
+
+
+
+
+// 2. Sorts the array :
+
+		//sort();
+
+// example
+
+var fruits = ["Orange", "Apple", "Banana"];
+
+fruits.sort();
+
+
+
+
+// 3. Joins arrays and returns an array with the joined arrays :
+
+		//concat();
+
+// example
+
+var fruits = ["Orange", "Apple", "Banana"];
+
+fruits.concat();
+
+
+
+
+// 4. Joins all elements of an array into a string :
+
+		//join();
+
+// example
+
+var fruits = ["Orange", "Apple", "Banana"];
+
+fruits.join();
```

Comment:
You are using the join method with nothing, not helpful for the learner. You should add the output also.